tests/tests.yaml

```diff
@@ -1,4 +1,10 @@
+# Bundle level test. Only execute the 20 and 30 class series of tests
+tests: "*"
+# Adding tims PPA so we get current dependencies
+sources:
+  - ppa:tvansteenburgh/ppa
```

Comment:
If the tyou st is executed in charmbox this won't be necessary. Not a nack but just FYI